### PR TITLE
WRQ-404: Updated `lerna` and `chalk` major dependency

### PR DESCRIPTION
**Pull request by andreitirlalge**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
`lerna` v.7 and `chalk` v.5 has been released.
This modules needs to be updated to the latest node modules to fix security vulnerabilities.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated `lerna` and `chalk` module to the latest version.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
There were breaking changes affecting this repository.
[LERNA]
- After updating we strongly recommend running `lerna repair` in your project. This will migrate your lerna.json to the latest and greatest and remove any outdated options. ( https://github.com/lerna/lerna/releases?page=2 )
- Replaced `lerna bootstrap` with `npm install`. ( https://lerna.js.org/docs/legacy-package-management )
- `lerna init` can no longer be run on an existing lerna repo
- node v14 support is dropped because it is end of life

[CHALK]
- The package is now pure ESM (https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c)

### Links
[//]: # (Related issues, references)
WRQ-404
https://github.com/lerna/lerna/releases?page=2
https://lerna.js.org/docs/legacy-package-management
https://github.com/chalk/chalk/releases
https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

### Comments
Enact-DCO-1.0-Signed-off-by: Andrei Tirla andrei.tirla@lgepartner.com

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]

File: packages/theme/index.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 const fs = require('fs');
 const path = require('path');
```
Comment (hong6316) [inline packages/theme/index.js:2]:
```suggestion
const path = require('path');
let chalk;
import('chalk').then(({default: _chalk}) => {
	chalk = _chalk;
});
```

File: packages/theme/index.js [hunk 2]
```diff
@@ -48,39 +47,41 @@ module.exports = {
 		});
 	},
 	complete: ({directory, name}) => {
```
Comment (hong6316) [inline packages/theme/index.js]: revert `complete:`

**Review by hong6316 — COMMENTED** [review]
Too many diff lines are occurred.
Could you test & verify those suggested changes?

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**